docs/wallets/wallets/ledger.md

```diff
@@ -0,0 +1,142 @@
+---
+id: ledger
+hide_title: true
+sidebar_label: Ledger
+slug: /ledger
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Ledger Hardware Wallet
+
+<img src={useBaseUrl("img/wallets/ledger/ledger-header.jpg")} />
```

Comment:
We've been leaving these out to start as Pete will likely be doing something a little different this time around.

Reply:
Gotcha. Thanks for the heads up. I've removed it.